test/hotspot/jtreg/serviceability/attach/AttachSetGetFlag.java

```diff
@@ -57,11 +57,6 @@ public static void main(String... args) throws Exception {
     testGetFlag("MaxHeapFreeRatio", "60");
     testSetFlag("MaxHeapFreeRatio", "50", "60");
 
-    // Test a non-manageable size_t flag.
-    // Since it is not manageable, we can't test the setFlag functionality.
-    testGetFlag("ArrayAllocatorMallocLimit", "128");
-    // testSetFlag("ArrayAllocatorMallocLimit", "64", "128");
-
```

Comment:
You need to replace this with another non-manageable size_t flag so that code coverage is maintained.

Reply:
Fixed.